core/src/main/java/io/th0rgal/oraxen/utils/drops/Drop.java

```diff
@@ -153,7 +151,7 @@ public void spawns(Location location, ItemStack itemInHand) {
 
     public void furnitureSpawns(Entity baseEntity, ItemStack itemInHand) {
         ItemStack baseItem = OraxenItems.getItemById(sourceID).build();
-        Location location = baseEntity.getLocation();
+        Location location = BlockHelpers.toBlockLocation(baseEntity.getLocation());
```

Comment:
Wondering if BlockHelpers#toBlockCenterLocation is more accurate here
Think block location will sometimes spit it out into walls or something

Reply:
The implementation of toBlockCenterLocation also exists in dropItemNaturally, so we do not need to use toBlockCenterLocation
![image](https://github.com/oraxen/oraxen/assets/8757536/a419d045-ed3e-406e-b593-1ea076648374)